### ISSUE TITLE
Hjson

### ISSUE THREAD

**Issue by ashifolfi** [JujuAdams/SNAP#41]
https://hjson.github.io/

I would like to see support for Hjson for SNAP. Hjson is a more human friendly  They have various implementations which you could reference and it does share a lot of it's syntax with standard JSON so that might assist in implementing this.

Having a format like this that could be used to load in like translations (tModLoader uses Hjson for language files, as a real world example of it's usage), dialog, etc. would be nice, YAML isn't quite my thing and JSON is too strict for my liking.

There's a huge bunch of files you could use to test present in the official implementations. Here's a link to the ones present in the C++ implementation: https://github.com/hjson/hjson-cpp/tree/master/test/assets

A full implementation would be nice but I won't complain if it lacks a few things here and there. I'd be happy even if it only supported reading.

**Comment by JujuAdams** [comment]
I agree that this would be a good addition; however, I have zero time to write a parser at the moment.

All is not lost: you may want to check out some [JSON alternatives](https://github.com/JujuAdams/ExtendingJSON) that I wrote because I ran into the same usage concerns that you're sharing. They're not quite as fancy as Hjson but they're close.

**Comment by ashifolfi** [comment]
> I agree that this would be a good addition; however, I have zero time to write a parser at the moment.
> 
> All is not lost: you may want to check out some [JSON alternatives](https://github.com/JujuAdams/ExtendingJSON) that I wrote because I ran into the same usage concerns that you're sharing. They're not quite as fancy as Hjson but they're close.

👍 these do look like they could fill the specific niche I planned to use hjson for in the meantime! I myself am not familiar enough with GML right now to be comfortable writing a full parser but in the future if it's still up for grabs I might try!
